society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -2,6 +2,7 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="chicago" default-locale="en-US">
   <info>
     <title>Society of Biblical Literature 2nd edition (full note)</title>
+    <title-short>SBL</title-short>
```

Comment:
Wouldn't it make sense to add the edition? Maybe change it to "SBL (2nd edition)"